### ISSUE TITLE
Make image descriptions more visible in accessibility metadata display

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#474]
While reviewing the second draft of the Accessibility Metadata Display Guide 2.0, we noticed that alternative descriptions for images aren't explicitly visible in the key information sections. Currently, this important accessibility feature is embedded within the broader "Supports nonvisual reading" category.

The presence of alternative descriptions is crucial information both for end users and as recognition of the significant work publishers put into creating them - it's one of the most important and time-consuming tasks when producing accessible publications.

We propose expanding the scope of the current "Charts, diagrams, math, and formulas" section, which is currently quite limited and STEM-focused content. We could rename it to "Non-textual content" (or find a better title) and use it to communicate the presence of:
- Short alternative descriptions for images
- Extended descriptions for complex images 
- Accessible mathematical and chemical formulas
- Charts and diagram descriptions

**Comment by clapierre** [comment]
I love this idea Gregorio!

**Comment by gautierchomel** [comment]
Editor's proposes "visual and complex contents".

**Comment by GeorgeKerscher** [comment]
Closing because image alt text and extended descriptions have been elevated. If something is still missing, please file a new issue.
